app.js

```diff
@@ -147,7 +147,11 @@ app.use(lessMiddleWare({
 }));
 
 app.use( express.static(tmpDir));
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(function(req, res, next) {
```

Comment:
I'd put it in a `utils` lib, myself. Wherever you're housing your universal error handler, etc is a good place